MultiplatformDemoWithSync/shared/src/commonMain/kotlin/io/realm/kotlin/demo/model/CounterRepository.kt

```diff
@@ -41,7 +42,7 @@ class CounterRepository {
     private var realm: Realm
     private val counterObj: Counter
 
-    private val app: App = App.create(AppConfiguration.Builder(MONGODB_REALM_APP_ID).build())
+    private val app: App = App.create(AppConfiguration.Builder(MONGODB_REALM_APP_ID).log(LogLevel.ALL).build())
```

Comment:
We probably need to add a comment about `partitionValue = "demo-parition"` option. If this is set here and defined server side then `realm_id` value should be removed (otherwise sync will fail to reconcile the generated partition key with the hard coded one)

Reply:
I removed the partition value from the model 👍